src/MockClassifier.Api/Services/NaturalLanguageService.cs

```diff
@@ -0,0 +1,25 @@
+﻿using MockClassifier.Api.Models;
+using MockClassifier.Api.Interfaces;
+
+namespace MockClassifier.Api.Services
+{
+    public class NaturalLanguageService : IClassifierService
+    {
+
+        public string[] Classify(string messageBody)
+        {
+            var token = "";
+            switch (messageBody)
```

Comment:
Would it be safer to cast this to lowercase and change the case comparisons to lower too? It would eliminate people entering things in the wrong case when demonstrating